### ISSUE TITLE
Stroked circles are not closed.

### ISSUE THREAD

**Issue by icefoxen** [nical/lyon#177]
Full issue description is here: https://github.com/ggez/ggez/issues/161

Basically, I create a circle as below:

```rust
    use lyon::tessellation as t;
    pub fn circle(&mut self,
                  mode: DrawMode,
                  point: Point2,
                  radius: f32,
                  tolerance: f32)
                  -> &mut Self {
        {
            let buffers = &mut self.buffer;
            match mode {
                DrawMode::Fill => {
                ...
                }
                DrawMode::Line(line_width) => {
                    let builder = &mut t::BuffersBuilder::new(buffers, VertexBuilder);
                    let options = t::StrokeOptions::default()
                        .with_line_width(line_width)
                        .with_tolerance(tolerance);
                    t::basic_shapes::stroke_circle(t::math::point(point.x, point.y),
                                                   radius,
                                                   &options,
                                                   builder);
                }
            };
        }
        self
    }
```

And when drawing it I get this:

![image](https://user-images.githubusercontent.com/1335133/30786671-b30d0d38-a147-11e7-82cc-83474c89d2e7.png)

Filled circles and other shapes appear to work as expected.

**Comment by nical** [comment]
Oops! Thanks for reporting this. I fixed it in #178 which will be in 0.8. I can release it any time, there's already been plenty of cool stuff since 0.7.x. Let me know if it's important for you to have this fix in a 0.7.x release. That would take a little bit longer to sort out because I think that I messed up the history of the 0.7 branch and I want to make sure I am not erasing whatever I put in 0.7.4 when releasing a 0.7.5.
